src/OrchardCore.Modules/OrchardCore.Seo/GraphQL/SeoMetaInputObjectType.cs

```diff
@@ -0,0 +1,22 @@
+﻿using GraphQL.Types;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Apis.GraphQL.Queries;
+using OrchardCore.Seo.Models;
+
+namespace OrchardCore.Seo.GraphQL;
+
+public class SeoMetaInputObjectType : WhereInputObjectGraphType<SeoMetaPart>
```

Comment:
Isn't it used for querying? There shouldn't be any filtering on this. So probably remove it.